civis/civis-aqueduct-utils/civis_aqueduct_utils/share.py

```diff
@@ -67,3 +70,41 @@ def main():
 
 if __name__ == "__main__":
     main()
+
+
+# Function to load file onto GitHub
+# Works with CSVs, HTML, but not parquet?
+def upload_file_to_github(TOKEN, REPO, BRANCH, PATH, local_file_path, commit_message):
```

Comment:
Also, can we add a docstring documenting how this function is intended to be used?